zcashd/entrypoint.sh

```diff
@@ -34,6 +34,7 @@ if [[ -n "${ZCASHD_ZSHIELDCOINBASE}" ]];then ZCASHD_CMD+=" -zshieldcoinbase=${ZC
 if [[ -n "${ZCASHD_RPCUSER}" ]];then ZCASHD_CMD+=" -rpcuser=${ZCASHD_RPCUSER}";fi
 if [[ -n "${ZCASHD_RPCPASSWORD}" ]];then ZCASHD_CMD+=" -rpcpassword=${ZCASHD_RPCPASSWORD}";fi
 if [[ -n "${ZCASHD_RPCBIND}" ]];then ZCASHD_CMD+=" -rpcbind=${ZCASHD_RPCBIND}";fi
+if [[ -n "${ZCASHD_RPCPORT}" ]];then ZCASHD_CMD+=" -rpcbind=${ZCASHD_RPCPORT}";fi
```

Comment:
should this pass `ZCASHD_RPCPORT` to `-rpcport` rather than `-rpcbind`?

Reply:
Yes :man_facepalming: 

Fixed